src/app/sign-in/page.tsx

```diff
@@ -0,0 +1,81 @@
+export default function LogIn() {
+  return (
+    <main>
+      <div>
+        <div className={styles.parentContainer}>
+          <div className={styles.mainContainer}>
+            <div className={styles.imageContainer}>
+              <img src="eclipse-image.png" className={styles.image} />
+            </div>
+            <div className={styles.formOuterContainer}>
+              <div className={styles.formInnerContainer}>
+                <h1 className={styles.signInText}>SIGN IN</h1>
+                <h3 className={styles.signUpText}>
+                  Don't have an account yet?{" "}
+                  <a href="/signup" className={styles.signUpRefText}>
+                     Sign up!
+                  </a>
+                </h3>
+                <form className={styles.form}>
+                  <div className="w-full">
+                    <label className={styles.label} htmlFor="username">
+                      Username
+                    </label>
+                    <input
+                      className={styles.input}
+                      id="username"
+                      type="text"
+                      placeholder="username"
+                    />
+                  </div>
+                  <div className="w-full">
+                    <label className={styles.label} htmlFor="password">
+                      Password
+                    </label>
+                    <input
+                      className={styles.input}
+                      id="password"
+                      type="text"
```

Comment:
quick improvement, if you [change the type to `password`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password) instead of `text` then it should obscure the password with dots